js/src/tooltip.js

```diff
@@ -55,7 +55,31 @@ const Tooltip = (() => {
     TOP    : 'top',
     RIGHT  : 'right',
     BOTTOM : 'bottom',
-    LEFT   : 'left'
+    LEFT   : 'left',
+    TOPLEFT    : 'top',
+    TOPRIGHT   : 'top',
+    RIGHTTOP   : 'right',
+    RIGHTBOTTOM: 'right',
+    BOTTOMLEFT : 'bottom',
+    BOTTOMRIGHT: 'bottom',
+    LEFTTOP    : 'left',
+    LEFTBOTTOM : 'left'
```

Comment:
I don't understand why you didn't use Popper.js placement with `-start` or `-end` see : https://popper.js.org/popper-documentation.html#Popper.placements

Reply:
I had the same question from @FezVrasta 
Please see https://github.com/FCOO/bootstrap-popover-extensions/issues/1 for the differents

Reply:
Depending on what locate a person lives (i.e. with a LTR language or RTL language), strt and end could be opposite to what the user thinks.

For users whose written language is LTR (left to right), start is on the left, while those users whose language is RTL (right to left), start is on the right.  In my opinion start and end are ambiguous, depends on your written language direction.

Left and Right are more concrete values (left is always left, and right is always right, but start is not always on the left as it could be on the right, depending on perspective).

Reply:
But then you have to handle the additional logic of converting "left" to "top" when a tooltip changes its placement from top to let.

top-start -> left-start

Vs

top-left -> left-top

That's the main reason why Popper.js uses start and end